ui/raidboss/data/05-shb/trial/wol-ex.js

```diff
@@ -768,8 +798,7 @@ const translate = (data, obj) => {
         'Imbued Ice/Fire': '魔法剣 (氷／火)',
         'Imbued Stone': '魔法剣 (土)',
         'Katon\\: San': '火遁の術：参',
-        '(?<! )Limit 1/3': '幻光召喚フェイス 1/3',
-        '(?<! )Limit 2/3': '幻光召喚フェイス 2/3',
+        '(?<! )Limit(?! Break)': '幻光召喚フェイス',
```

Comment:
I am not very sure about this translation I have added before.
Does it tell the phase change, or just a plain skill (limit break) that boss uses?

Reply:
It is for these lines, which I have changed in this patch: https://github.com/quisquous/cactbot/blob/394faba819e4399e70e058f65bb27a3934cabb61/ui/raidboss/data/05-shb/trial/wol-ex.txt#L107

e.g. "Limit -> BLM/WHM"

It used to say "Limit 2/3 -> BLM/WHM" but I found out it can be any limit break, so I removed the 2/3.  Sorry if I made the translation bad in the process.

Reply:
Oh, so it means the fake skill that matches "To the Limit" in the random summons?
If true, then it should be translated as "リミットチャージ", which is the skill "To the Limit" written in Japanese client.